main.ts

```diff
@@ -13,64 +16,85 @@ function main() {
   Deno.serve({ port: PORT }, handle);
 }
 
-/**
- * handle handles an incoming HTTP request.
- */
 async function handle(request: Request): Promise<Response> {
   const url = new URL(request.url);
 
   switch (true) {
     case request.method === "GET" && url.pathname === "/": {
       const body = Deno.readTextFileSync("static/index.html");
+      console.log("Found index.html");
```

Comment:
shouldn't all of these be removed?

Reply:
> shouldn't all of these be removed?

Good question! I was originally considering leaving them for the ability to see how the server works as a learning tool. What do you think, @oscisn93? Another option is to properly demo [Deno’s standard logger](https://deno.land/std/log/mod.ts).

Reply:
Yeah I think the standard logger is a great idea. And a good teaching moment

Reply:
Thanks for your input! Here's what I did: <https://github.com/acmcsufoss/api-workshop/pull/2/commits/db1e53f808143aff4b1bdc95c30906cc31fc7d48>.